server/lib/sanitize/sanitize.js

```diff
@@ -46,6 +46,11 @@ const optionsNames = new Set([ 'nonJsonBody' ])
 const sanitizeParameter = (input, name, config, place, res) => {
   const parameter = getParameterFunctions(name, config.generic)
 
+  if (parameter.drop) {
+    delete input[name]
+    return
```

Comment:
a little test on drop in `tests/unit/libs/046-sanitize.js` would make me happy  [non-blocking]

Reply:
That's tested through the `ignore` generic parameter (see `it('should accept an ignored generic parameters'`). I started writing a more specific test, with something like:
```js
const configs = {
  name: {
    drop: true
  }
}
```
but that won't do, as that's the parameter controller-specific configuration and not the parameter global settings: we would need to have a parameter in `server/lib/sanitize/parameters.js` using `drop: true` to test that... which is what the `ignore`  generic does